python/fledge/common/logger.py

```diff
@@ -76,19 +82,27 @@ def _get_process_name():
 
     logger = logging.getLogger(logger_name)
 
+    # if no destination is set, use the fledge default
+    if destination == None:
+        destination = default_destination
+
     if destination == SYSLOG:
         handler = SysLogHandler(address='/dev/log')
     elif destination == CONSOLE:
-        handler = logging.StreamHandler(sys.stdout)
+        handler = logging.StreamHandler(sys.stderr)
     else:
         raise ValueError("Invalid destination {}".format(destination))
 
-    # TODO: Consider using %r with message when using syslog .. \n looks better than #
     process_name = _get_process_name()
+    # TODO: Consider using %r with message when using syslog .. \n looks better than #
     fmt = '{}[%(process)d] %(levelname)s: %(module)s: %(name)s: %(message)s'.format(process_name)
     formatter = logging.Formatter(fmt=fmt)
+
     handler.setFormatter(formatter)
-    logger.setLevel(level)
-    logger.propagate = propagate
+    if level != None:
```

Comment:
`if level is not None:`  or just `if level`

Reply:
explicitly checking against None to differentiate between "no level provided" and "level set to zero" cases.